src/components/RelatedImages.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="photo_related-images" v-if="relatedImages && relatedImages.length > 0">
-    <h3>Related Images</h3>
+    <h3 style="margin-left:10px">Related Images</h3>
```

Comment:
The ticket says that the grid has a wrong margin, not the header. This should be fixed in the image grid

Reply:
@brenoferreira I have made the changes ,please review my pr

Reply:
@brenoferreira Thanks for reviewing my pr. I have made the requested changes, thanks.